apps/storybook/src/WelcomeMessage.stories.tsx

```diff
@@ -9,11 +9,17 @@ export default {
   decorators: [MessageLayoutDecorator],
 };
 
-export const Normal = () => (
+export const Variant1 = () => (
```

Comment:
I don't like this naming. Can we be more specific? Something that says InviteScriptWelcomeTemplate or something?